packages/deno/src/generators/utils/ts-config.ts

```diff
@@ -0,0 +1,15 @@
+import { Tree } from '@nrwl/devkit';
+
+// TODO(caleb): switch to @nrwl/js version once we update and make it a dep
```

Comment:
Might be fine to leave it TBH. Need to re-think how platform agnostic `@nrwl/js` is since it assumes Node and npm for a lot of things.